cmd/kogito/command/install/trusty.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package install
+
+import (
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/converter"
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/flag"
+
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/context"
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/shared"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/spf13/cobra"
+)
+
+type installTrustyFlags struct {
+	flag.InstallFlags
+	flag.InfinispanFlags
+	flag.KafkaFlags
+}
+
+type installTrustyCommand struct {
+	context.CommandContext
+	command *cobra.Command
+	flags   installTrustyFlags
+	Parent  *cobra.Command
+}
+
+func initInstallTrustyCommand(ctx *context.CommandContext, parent *cobra.Command) context.KogitoCommand {
+	cmd := &installTrustyCommand{
+		CommandContext: *ctx,
+		Parent:         parent,
+	}
+
+	cmd.RegisterHook()
+	cmd.InitHook()
+
+	return cmd
+}
+
+func (i *installTrustyCommand) Command() *cobra.Command {
+	return i.command
+}
+
+func (i *installTrustyCommand) RegisterHook() {
+	i.command = &cobra.Command{
+		Use:     "trusty [flags]",
+		Short:   "Installs the Kogito Trusty Service in the given Project",
+		Example: "trusty -p my-project",
+		Long: `'install trusty' will deploy the Trusty service to enable capturing tracing events produced by one or more Kogito services and provide analysis capabilities on top of the data.
```

Comment:
maybe add a external reference about it to redirect user to the correct place to learn more about trusty AI service?

Reply:
Hi @spolti , thank you for the comments! Since the trusty ai initiative is still under strong development we don't have a very well defined documentation. I've created a small pull request to add a `README` in the kogito-apps https://github.com/kiegroup/kogito-apps/pull/347 (the e2e example would point to this example https://github.com/kiegroup/kogito-examples/pull/315 for the time being). If it makes sense to you, I can link to that readme as soon as we merge it. lmkwyt

Reply:
I am ok adding the readme reference here, let's see what the others think about it.

Reply:
Hi @spolti , I've added a link to the trusty readme that is going to be added by this https://github.com/kiegroup/kogito-apps/pull/347